<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<!--
Design by Free CSS Templates
http://www.freecsstemplates.org
Released for free under a Creative Commons Attribution 2.5 License

Name       : Communication
Description: A two-column, fixed-width design with dark color scheme.
Version    : 1.0
Released   : 20090523

-->
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta name="keywords" content="" />
<meta name="description" content="" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<title>AMBE 101 :: Lecture 11</title>
<script type="text/javascript" src="js/mootools.svn.js"></script>


<!-- Mootools -->

<link href="stylec.css" rel="stylesheet" type="text/css" media="screen" />
<style type="text/css">
<!--
.button{
	background:url(http://eagri.org/eagri50/AMBE101/button.png)  -32px right no-repeat;
	color:#FFFFFF;
	clear:both;
	display:block;
	float:left;
	font-size:13px;
	font-weight:bold;
	height:31px;
	line-height:31px;
	width:250px;
	margin-right:30px;
}
	a.button {
		text-decoration:none;
	}
	.button span {
		background:url(images/button.jpg) left top no-repeat;
		display:block;
		height:30px;
		width:250px;
		line-height:31px;
		padding-left:35px;
		padding-right:8px;
		margin-right:20px;
	}
.v-menu{
	
	width:250px;
	clear:both;
}
	ul.v-menu, .v-menu li{
		padding:0; 
		margin:0;
		line-height: normal;
		list-style:none;
	}
	ul.v-menu{
		clear:both;
		margin-top:6px;
		padding:6px 5px;
	}
		.v-menu li a{
			color:#555555;
			font-weight:bold;
			display:block;
			border-top:solid 1px #DEDEDE;
			
			text-decoration:none;
		}
		.v-menu li a:hover{
			color:#999999;
		}
.style1 {
	font-size: 14px;
	color: #FFFFFF;
}
body {
	background-color: #FFFFFF;
}
body,td,th {
	color: #333333;
}
.style4 {color: #FFFFFF}
-->
</style>
<script type="text/javascript">
		window.addEvent('domready', function(){
			//-vertical
			
			var mySlide = new Fx.Slide('v-menu2');
			mySlide.hide(); 
			$('toggle').addEvent('click', function(e){
				e = new Event(e);
				mySlide.toggle();
				e.stop();
			});

		}); 
	</script>
</head>
<body>
<div id="wrapper">
	<div id="header">
		<div id="logo">
			<h1>&nbsp;	</h1>
	  </div>
	</div>
	<!-- end #header -->
  <!-- end #menu -->
  <div id="page">
	<div id="page-bgtop">
	<div id="page-bgbtm">
		<div id="content">
			<div class="post">
			<div class="post-bgtop">
			<div class="post-bgbtm">
				<h2 align="center" class="titlen style1"><strong><span class="style4">AMBE 101 :: Lecture 11 :: </span></strong><strong>GENE EXPRESSION</strong></h2>
				<p align="right"><img src="images/back.jpg" alt="" width="167" height="30" usemap="#Map" />
                  <map name="Map" id="Map2">
                    <area shape="rect" coords="1,1,168,30" href="index.html" alt="" />
                  </map>
				</p>
				<p>Is the process by which information from a <a href="http://en.wikipedia.org/wiki/Gene" title="Gene">gene</a> is used in the synthesis of a functional <a href="http://en.wikipedia.org/wiki/Gene_product" title="Gene product">gene product</a>. These products are often <a href="http://en.wikipedia.org/wiki/Protein" title="Protein">proteins</a>, but in non-protein coding genes such as <a href="http://en.wikipedia.org/wiki/RRNA" title="RRNA">rRNA</a> genes or <a href="http://en.wikipedia.org/wiki/TRNA" title="TRNA">tRNA</a> genes, the product is a functional <a href="http://en.wikipedia.org/wiki/RNA" title="RNA">RNA</a>. The process of gene expression is used by  all known life - <a href="http://en.wikipedia.org/wiki/Eukaryotes" title="Eukaryotes">eukaryotes</a> (including <a href="http://en.wikipedia.org/wiki/Multicellular_organisms" title="Multicellular organisms">multicellular organisms</a>), <a href="http://en.wikipedia.org/wiki/Prokaryotes" title="Prokaryotes">prokaryotes</a> (<a href="http://en.wikipedia.org/wiki/Bacteria" title="Bacteria">bacteria</a> and <a href="http://en.wikipedia.org/wiki/Archaea" title="Archaea">archaea</a>) and <a href="http://en.wikipedia.org/wiki/Virus" title="Virus">viruses</a> - to generate the <a href="http://en.wikipedia.org/wiki/Macromolecule" title="Macromolecule">macromolecular</a> machinery for life. Several steps in the  gene expression process may be modulated, including the <a href="http://en.wikipedia.org/wiki/Transcription_(genetics)" title="Transcription (genetics)">transcription</a>, <a href="http://en.wikipedia.org/wiki/RNA_splicing" title="RNA splicing">RNA  splicing</a>, <a href="http://en.wikipedia.org/wiki/Translation_(biology)" title="Translation (biology)">translation</a>, and <a href="http://en.wikipedia.org/wiki/Post-translational_modification" title="Post-translational modification">post-translational modification</a> of a protein. <a href="http://en.wikipedia.org/wiki/Regulation_of_gene_expression" title="Regulation of gene expression">Gene regulation</a> gives the <a href="http://en.wikipedia.org/wiki/Cell_(biology)" title="Cell (biology)">cell</a> control over structure and function, and is  the basis for <a href="http://en.wikipedia.org/wiki/Cellular_differentiation" title="Cellular differentiation">cellular differentiation</a>, <a href="http://en.wikipedia.org/wiki/Morphogenesis" title="Morphogenesis">morphogenesis</a> and the versatility and adaptability of any <a href="http://en.wikipedia.org/wiki/Organism" title="Organism">organism</a>. Gene regulation may also serve as a  substrate for evolutionary change, since control of the timing, location, and  amount of gene expression can have a profound effect on the functions (actions)  of the gene in a cell or in a multicellular organism.<br />
                  <img width="484" height="340" src="lec11_clip_image002_0000.jpg" align="left" hspace="12" />In <a href="http://en.wikipedia.org/wiki/Genetics" title="Genetics">genetics</a>, gene expression is the most fundamental  level at which <a href="http://en.wikipedia.org/wiki/Genotype" title="Genotype">genotype</a> gives rise to the <a href="http://en.wikipedia.org/wiki/Phenotype" title="Phenotype">phenotype</a>. The genetic code is &quot;interpreted&quot;  by gene expression, and the properties of the expression products give rise to  the organism's phenotype.</p>
				<p><a href="http://eagri.org/eagri50/Transcription.htm"><strong>Transcription</strong></a><strong><u> (Video) </u></strong><br />
				  The gene itself is typically a long stretch  of <a href="http://en.wikipedia.org/wiki/DNA" title="DNA">DNA</a>. It is a blueprint for the production of  RNA. The production of RNA copies of the DNA is called <a href="http://en.wikipedia.org/wiki/Transcription_(genetics)" title="Transcription (genetics)">transcription</a>, and is performed by <a href="http://en.wikipedia.org/wiki/RNA_polymerase" title="RNA polymerase">RNA polymerase</a>, which adds one RNA <a href="http://en.wikipedia.org/wiki/Nucleotide" title="Nucleotide">nucleotide</a> at a time to a growing RNA strand. This RNA  is <a href="http://en.wikipedia.org/wiki/Complementarity_(molecular_biology)" title="Complementarity (molecular biology)">complementary</a> to the template 3' → 5' DNA strand,<a href="http://en.wikipedia.org/wiki/Gene_expression#cite_note-0">[1]</a> which is itself complementary to the coding 5' → 3' DNA  strand. Therefore, the resulting 5' → 3' RNA strand is identical to the coding DNA  strand with the exception that <a href="http://en.wikipedia.org/wiki/Thymine" title="Thymine">thymines</a> are replaced with <a href="http://en.wikipedia.org/wiki/Uracil" title="Uracil">uracils</a> in the RNA. A coding DNA strand reading  &quot;ATG&quot; is transcribed as &quot;UAC&quot; in RNA.<br />
                </p>
				<p>&nbsp;</p>
				<p><a href="http://eagri.org/eagri50/AMBE101/Videos/Translation.mov"><strong>Translation (Video)</strong></a><strong> </strong></p>
				<ol>
				  <li><strong>Initiation</strong><strong> </strong></li>
				  <li>30S  initiates binding to mRNA. </li>
				  <li>locates  Shine-Dalgarno sequence (3-9 bases near 5' end of mRNA). </li>
				  <li>ribosome  finds first AUG codon. </li>
				  <li>50S  ribosome binds. </li>
				  <li>tRNA  carries N-formylmethione to first position </li>
				  <li><strong>Elongation</strong><strong> </strong></li>
				  <li>2  adjacent sites on ribosome: P  and A site.A site  accepts a new tRNA-AA.Psite holds existing chain peptide transferred from P site tRNA to A-site AA </li>
				  <li>enzyme  activity is in ribosomal RNA, not protein </li>
				  <li>also  required: Energy (GTP) and elongation factors </li>
				  <li><strong>Termination</strong> </li>
				  <li>reach  a &quot;stop codon&quot; UAG, UAA, or UGA </li>
				  <li>no  t-RNAs for release, but release factors required </li>
				  <li>Net  cost: 4 phosphate bonds/amino  acid added! </li>
				  <li>B. Genetic Code </li>
				  <ol>
				    <li>AUG  = universal &quot;start&quot; codon</li>
				    <li>UAG,  UAA, UGA = &quot;stop&quot; codons</li>
				    <li>A  few messages in bacteria use GUG as start, but still need Shine-Dalgarno  sequence, stil code for N-formylmethionine.</li>
			      </ol>
			    </ol>
				<p><a href="http://eagri.org/eagri50/AMBE101/Videos/Lacoperon.mov"><strong>Classic  example: The lac operon (Video)</strong></a><strong> </strong><br />
                <img border="0" width="507" height="284" src="lec11_clip_image004_0000.jpg" /> </p>
				<ol>
				  <li>gene  is regulated by negative control;  in absence of specific repressor, gene is transcribed just like constitutive  gene. In order to regulate, must add specific block. Must say &quot;no&quot;; otherwise gene is not  down-regulated.</li>
				  <li><strong>Lactose</strong> = milk sugar, disaccharide made of galactose +  glucose. In order to metabolize lactose, cells must produce enzyme  ß-galactosidase, split lactose into galactose + glucose </li>
				  <li><strong>Observation</strong>: add lactose to cells: within minutes, ß-galactosidase enzyme appears, also lac permease in membrane, and a third  protein, transacetylase. Level  of ß-galactosidase enzyme can accumulate to level of 10% of cytoplasmic protein</li>
				  <li><strong>Explanation:</strong> in absence of lactose (= inducer), lac repressor blocked operator site.</li>
				  <li>Lac  repressor is allosteric protein. Coded for by another region of DNA  (constitutive gene, weak promoter, low level of expression)</li>
				  <li>Effector molecule = Inducer  is lactose (actually allolactose, or analog such as IPTG) binds to repressor  protein, repressor released, RNA is made, all genes turned on as unit</li>
			    </ol>
				<p>Some genes are regulated by activator proteins</p>
				<ol>
				  <li>genes  with weak promoters are rarely transcribed.some such genes can be activated by activator protein, causes RNA  polymerase to bind more tightly.often there are two components to such  regulation: a sensor protein and an activator protein. The activator protein is  inactive until phosphorylated by the sensor, then it activates transcription,  gene product is made.</li>
			    </ol>
				<p><em>Lac</em> operon – in presence of inducer (  lactose)</p>
				<ol>
				  <li><img border="0" width="557" height="344" src="lec11_clip_image006_0000.jpg" /> </li>
			    </ol>
				<p><em>Lac</em> operon – in  absence of lactose<br />
                <img border="0" width="506" height="321" src="lec11_clip_image008_0000.jpg" /> </p>
				<p>Role of mRNA<br />
				  Carries coding information for amino acids = codons, 3 adajacent nucleotide bases Example: AAA,  AGU, etc.leader sequence on mRNA (called Shine-Dalgarno sequence) binds to  complementar sequence on small ribosome  subunit.<br />
  <img border="0" width="332" height="304" src="lec11_clip_image010_0000.jpg" /> <br />
				  Role of  ribosome <br />
				  acts as a &quot;decoding box&quot; or &quot;tape  player&quot; for the information in mRNA30S &amp; 50S subunits (= 70S)30S has  16S RNA + 21 proteins.50S has 23S &amp; 5S RNA +  34 proteins.<br />
				  Role of tRNA </p>
				
				<ul>
				  <li>structure:  4 loops, anticodon, AA binding site</li>
				  <li>~  60 types in bacteria (&gt;100 in mammals)</li>
				  <li>only  73-93 nucleotides long</li>
				  <li>some  modified bases: pseudouridine, inosine, others</li>
				  <li>modified  after transcription</li>
				  <li>extensive  hairpin loops</li>
				  <li>anticodon  site: recognizes codon on mRNA</li>
				  <li>AA  added by enzyme: AA-tRNA activating enzymes</li>
				  <li>ATP  required, forms AA-AMP + PP, then AA-tRNA + AMP</li>
			    </ul>
				<table width="27%" border="0">
                  <tr>
                    <td><strong><a href="pdf/lec32.pdf"><img src="images/arlogo.png" alt="" width="32" height="32" border="0" /></a></strong></td>
                    <td bgcolor="#A20807"><strong><a href="pdf/lec11.pdf" class="style4">Download this lecture as PDF here</a></strong></td>
                  </tr>
                </table>
				<br />
				<p><img src="images/back.jpg" alt="" width="167" height="30" usemap="#Map" />
				  <map name="Map" id="Map">
				    <area shape="rect" coords="1,1,168,30" href="index.html" alt="" />
			      </map>
			    </p>
			</div>
			</div>
			</div>
			</div>
		<div style="clear: both;">&nbsp;</div>
	</div>
	</div>
	</div>
	<!-- end #page -->
</div>
	<div id="footer">
		<p>Copyright (c) 2011. TAMIL NADU AGRICULTURAL UNIVERSITY</p>
</div>
	<!-- end #footer -->
</body>
</html>
